src/libpit/util.c

```diff
@@ -7,7 +7,7 @@ char *getext(char *file) {
 
   ext = NULL;
 
-  for (i = sys_strlen(file)-1; i > 0; i--) {
+  for (i = sys_strlen(file)-1; i >= 0; i--) {
```

Comment:
This fixes a crash when attempting to get the file suffix for a file the begins with a dot (such as `.empty` or `.DS_Store`.